operator/src/main/java/oracle/kubernetes/weblogic/domain/model/AuxiliaryImage.java

```diff
@@ -38,17 +37,19 @@
   @EnumClass(ImagePullPolicy.class)
   private String imagePullPolicy;
 
-  @Description("The command for this init container. Defaults to `cp -R $AUXILIARY_IMAGE_PATH/* "
-      + "$AUXILIARY_IMAGE_TARGET_PATH`. This is an advanced setting for customizing the container command for copying "
-      + "files from the container image to the auxiliary image emptyDir volume. Use the `$AUXILIARY_IMAGE_PATH` "
-      + "environment variable to reference the value configured in `spec.auxiliaryImageVolumes.mountPath`, which "
-      + "defaults to \"/auxiliary\". Use '$AUXILIARY_IMAGE_TARGET_PATH' to refer to the temporary directory created by "
-      + "the operator that resolves to the auxiliary image's internal emptyDir volume.")
-  private String command;
+  @Description("The source location of the WebLogic Deploy Tooling installation within the auxiliary image. "
+          + "Defaults to '/auxiliary/weblogic-deploy'. If the value is set to 'None' or no files found at the default "
+          + "location, then the WebLogic Deploy Tooling installation copy is skipped. When specifying multiple "
+          + "auxiliary images, it is important to ensure that no more than one of the images "
+          + "supplies a WDT install home; if more than one WDT install home is copied, then domain deployment will "
+          + "fail.")
+  private String sourceWDTInstallHome;
 
-  @Description("The name of an auxiliary image volume defined in `spec.auxiliaryImageVolumes`. Required.")
-  @NotNull
-  private String volume;
+  @Description("The source location of the WebLogic Deploy Tooling model home within the auxiliary image. "
```

Comment:
that will be copied to '/aux/models' directory in the WebLogic Server container in all pods.